### PR TITLE
Refer to external openstack cloud controller manager

### PR DESCRIPTION
**Pull request by edisonxiang**
**What this PR does / why we need it**:
Refer to external openstack cloud controller manager in the main.go.
Adjust the pkg code directory.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -30,10 +30,11 @@ import (
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app/options"
 	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
-	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
-	_ "k8s.io/kubernetes/pkg/version/prometheus" // for version metric registration
+	_ "k8s.io/kubernetes/pkg/version/prometheus"        // for version metric registration
 	"k8s.io/kubernetes/pkg/version/verflag"
 
+	"github.com/dims/openstack-cloud-controller-manager/pkg/openstack"
```
Comment (dims) [inline main.go]: should be `git.openstack.org/openstack/openstack-cloud-controller-manager`

please see https://github.com/dims/openstack-cloud-controller-manager/blob/master/glide.yaml#L1

Comment (edisonxiang) [reply main.go]: OK, But it is a litter strange like this.

Comment (edisonxiang) [reply main.go]: It is better to update glide.yaml too.

Comment (dims) [reply main.go]: @edisonxiang the plan is to move to openstack git repo when we have the basic functionality working hence the request to use the openstack git url in the packaging

**Comment by edisonxiang** [comment]
@dims Done. PTAL.

**Comment by dims** [comment]
@edisonxiang can you please squash the two commits? LGTM

**Comment by edisonxiang** [comment]
@dims Done:)

**Comment by dims** [comment]
LGTM. thanks

**Pull request merged**